fab/inventory/staging

```diff
@@ -29,9 +29,6 @@ hqtouch0-staging.internal-va.commcarehq.org
 [redis]
 hqdb1-staging.internal-va.commcarehq.org
 
-[memcached]
```

Comment:
@esoergel was this needed during your migration, or was it just from copy/paste?